hunts/templates/tools.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% block page_content %}
+
+<div class="row justify-content-between" style="margin:10">
+    <div class="col-10">
+        <h1>Tools and References</h1>
+    </div>
+
+    <div class="col-10">
+        <h2>Word Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li>https://quipqiup.com/</li>
+            <li>https://nutrimatic.org/</li>
+            <li>https://www.onelook.com/</li>
+            <li>https://www.thewordfinder.com/anagram-solver/</li>
+            <li>https://www.highlightpress.com.au/</li>
+            <li>http://www.npinsker.me/puzzles/crossword</li>
+        </ul>
+    </div>
+
+    <div class="col-10">
+        <h2>Logic Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li>Ask Max</li>
```

Comment:
Grid Puzzles - https://en.wikipedia.org/wiki/Nikoli_(publisher)

Reply:
And a good reference list of logic puzzles with example pictures and rules is: http://www.cross-plus-a.com/puzzles.htm